added vt1 option, modified date string format

jsturdy opened the pull request:
* added VT1 setting option in ```trimChamber```
* updated zero VFATs to use new function in ```cmsgemos```
* updated the time/date format to be everywhere ```strftime("%Y.%m.%d.%H.%M")```

cbravo135 commented on confChamber.py:
```diff
@@ -23,7 +23,7 @@
 from ROOT import TFile,TTree
 import subprocess,datetime
-startTime = datetime.datetime.now().strftime("%d.%m.%Y-%H.%M.%S.%f")
+startTime = datetime.datetime.now().strftime("%Y.%m.%d-%H.%M")
```
This doesn't seem to be like the others. This is closer to what I would pick.

jsturdy replied:
What's the difference?

cbravo135 replied:
here it is %Y first

jsturdy commented:
Is that not what I changed everything to?  Ah, I see, hyphen vs. dash, well, I kept the old format and just removed the trailing seconds, and ordered as you had mentioned yesterday

cbravo135 commented:
"%d.%m.%Y-%H.%M" -> "%Y.%m.%d.%H.%M" the bigger difference is switching %Y and %d

cbravo135 commented on VFATSCurveTools.py:
```diff
@@ -670,7 +670,7 @@ def runScanRoutine(self,vfat,debug=False):
 
     import subprocess,datetime
-    startTime = datetime.datetime.now().strftime("%d.%m.%Y-%H.%M.%S.%f")
+    startTime = datetime.datetime.now().strftime("%d.%m.%Y-%H.%M")
```
Here it is %d first

jsturdy replied:
Ah, got it

cbravo135 commented on pythonScript.py:
```diff
@@ -31,7 +31,7 @@
 
     import subprocess,datetime
-    startTime = datetime.datetime.now().strftime("%d.%m.%Y-%H.%M.%S.%f")
+    startTime = datetime.datetime.now().strftime("%d.%m.%Y-%H.%M")
```
Here it is %d first

cbravo135 commented on pythonScript_904.py:
```diff
@@ -25,7 +25,7 @@
 
     import subprocess,datetime
-    startTime = datetime.datetime.now().strftime("%d.%m.%Y-%H.%M.%S.%f")
+    startTime = datetime.datetime.now().strftime("%d.%m.%Y-%H.%M")
```
Here it is %d first

cbravo135 commented on run_scans.py:
```diff
@@ -16,19 +16,25 @@ def launchTestsArgs(tool, slot, link, chamber,vt2=0,perchannel=False,trkdata=Fal
     pass
 
-  startTime = datetime.datetime.now().strftime("%d.%m.%Y-%H.%M.%S.%f")
+  startTime = datetime.datetime.now().strftime("%Y.%m.%d-%H.%M")
```
Here it is %Y first

cbravo135 commented on setTRIMDAC.py:
```diff
@@ -30,7 +30,7 @@
 
     import subprocess,datetime
-    startTime = datetime.datetime.now().strftime("%d.%m.%Y-%H.%M.%S.%f")
+    startTime = datetime.datetime.now().strftime("%d.%m.%Y-%H.%M")
```
Here it is %d first

jsturdy commented:
Yeah, I didn't bother changing the scripts that aren't being used

cbravo135 commented on setTRIMDAC_and_Mask.py:
```diff
@@ -28,7 +28,7 @@
 
 import subprocess,datetime
-startTime = datetime.datetime.now().strftime("%d.%m.%Y-%H.%M.%S.%f")
+startTime = datetime.datetime.now().strftime("%d.%m.%Y-%H.%M")
```
Here it is %d first

cbravo135 commented on setTRIMDAC_and_Scurve.py:
```diff
@@ -30,7 +30,7 @@
 
     import subprocess,datetime
-    startTime = datetime.datetime.now().strftime("%d.%m.%Y-%H.%M.%S.%f")
+    startTime = datetime.datetime.now().strftime("%d.%m.%Y-%H.%M")
```
Here it is %d first

cbravo135 commented on setTRIMDAC_and_Threshold.py:
```diff
@@ -32,7 +32,7 @@
 
     import subprocess,datetime
-    startTime = datetime.datetime.now().strftime("%d.%m.%Y-%H.%M.%S.%f")
+    startTime = datetime.datetime.now().strftime("%d.%m.%Y-%H.%M")
```
Here it is %d first

cbravo135 commented on trimChamber.py:
```diff
@@ -43,7 +45,7 @@ def runCommand(cmd):
 from macros.fitScanData import fitScanData
 import subprocess,datetime
-startTime = datetime.datetime.now().strftime("%Y.%m.%d-%H.%M.%S.%f")
+startTime = datetime.datetime.now().strftime("%Y.%m.%d-%H.%M")
```
Here it is %Y first

cbravo135 commented on ultraScurve.py:
```diff
@@ -48,7 +48,7 @@
 
 import subprocess,datetime
-startTime = datetime.datetime.now().strftime("%d.%m.%Y-%H.%M.%S.%f")
+startTime = datetime.datetime.now().strftime("%Y.%m.%d-%H.%M")
```
Here it is %Y first

cbravo135 commented on ultraThreshold.py:
```diff
@@ -58,7 +58,7 @@
 
 import subprocess,datetime
-startTime = datetime.datetime.now().strftime("%d.%m.%Y-%H.%M.%S.%f")
+startTime = datetime.datetime.now().strftime("%Y.%m.%d-%H.%M")
```
Here it is %Y first

cbravo135 commented on ultra_QC2.py:
```diff
@@ -23,7 +23,7 @@
 
 import subprocess,datetime
-startTime = datetime.datetime.now().strftime("%d.%m.%Y-%H.%M.%S.%f")
+startTime = datetime.datetime.now().strftime("%d.%m.%Y-%H.%M")
```
Here it is %d first

cbravo135 commented on takeScurves.py:
```diff
@@ -11,7 +11,7 @@
 
 import subprocess,datetime
-startTime = datetime.datetime.now().strftime("%Y.%m.%d-%H.%M.%S.%f")
+startTime = datetime.datetime.now().strftime("%Y.%m.%d.%H.%M.%S.%f")
```
This one still has %S and %f
